dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -448,69 +451,51 @@ object LagomPlugin extends AutoPlugin {
-      Def.task {
-        val port = lagomCassandraPort.value
-        val cleanOnStart = lagomCassandraCleanOnStart.value
-        val classpath = (managedClasspath in Compile).value.map(_.data)
-        val jvmOptions = lagomCassandraJvmOptions.value
-        val maxWaiting = lagomCassandraMaxBootWaitingTime.value
-        val log = new SbtLoggerProxy(state.value.log)
-        Servers.CassandraServer.start(log, classpath, port, cleanOnStart, jvmOptions, maxWaiting)
-      }
-    } else {
-      Def.task {
-        val log = state.value.log
-        log.info(s"Cassandra won't be started because the build setting `${lagomCassandraEnabled.key.label}` is set to `false`")
-      }
-    }
+  private lazy val startCassandraServerTask = Def.task {
+    val port = lagomCassandraPort.value
+    val cleanOnStart = lagomCassandraCleanOnStart.value
+    val classpath = (managedClasspath in Compile).value.map(_.data)
+    val jvmOptions = lagomCassandraJvmOptions.value
+    val maxWaiting = lagomCassandraMaxBootWaitingTime.value
+    val log = new SbtLoggerProxy(state.value.log)
+    Servers.CassandraServer.start(log, classpath, port, cleanOnStart, jvmOptions, maxWaiting)
   }
 
-  private lazy val startKafkaServerTask = Def.taskDyn {
-    if ((lagomKafkaEnabled in ThisBuild).value) {
-      Def.task {
-        val log = new SbtLoggerProxy(state.value.log)
-        val zooKeeperPort = lagomKafkaZookeperPort.value
-        val kafkaPort = lagomKafkaPort.value
-        val kafkaPropertiesFile = lagomKafkaPropertiesFile.value
-        val classpath = (managedClasspath in Compile).value.map(_.data)
-        val jvmOptions = lagomKafkaJvmOptions.value
-        val targetDir = target.value
-        val cleanOnStart = lagomKafkaCleanOnStart.value
-
-        Servers.KafkaServer.start(log, classpath, kafkaPort, zooKeeperPort, kafkaPropertiesFile, jvmOptions, targetDir, cleanOnStart)
-      }
-    } else {
-      Def.task {
-        val log = state.value.log
-        log.info(s"Kafka won't be started because the build setting `${lagomKafkaEnabled.key.label}` is set to `false`")
-      }
-    }
+  private lazy val startKafkaServerTask = Def.task {
+    val log = new SbtLoggerProxy(state.value.log)
+    val zooKeeperPort = lagomKafkaZookeperPort.value
+    val kafkaPort = lagomKafkaPort.value
+    val kafkaPropertiesFile = lagomKafkaPropertiesFile.value
+    val classpath = (managedClasspath in Compile).value.map(_.data)
+    val jvmOptions = lagomKafkaJvmOptions.value
+    val targetDir = target.value
+    val cleanOnStart = lagomKafkaCleanOnStart.value
+
+    Servers.KafkaServer.start(log, classpath, kafkaPort, zooKeeperPort, kafkaPropertiesFile, jvmOptions, targetDir, cleanOnStart)
   }
 
+  private def sequential(tasks: Seq[Task[Closeable]]): Initialize[Task[Seq[Closeable]]] =
+    tasks.toList match {
+      case Nil => Def.task { Nil }
+      case x :: xs =>
+        Def.taskDyn {
+          val v = x.value
+          sequential(xs).map(v +: _)
+        }
+    }
```

Comment:
Small utility method to replace sbts `Def.sequential`. It's very similar, with primary difference is that this returns a sequence of the results of all the tasks, whereas sbt`s only returns the result of the last task executed.